CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 All notable changes to this project will be documented in this file.
 This project adheres to [Semantic Versioning](http://semver.org/).
 
+## [Unreleased]
+### Fixed
+- Fixes missing association admin class in datagrid filters.
+
```

Comment:
Don't touch this file directly please, we do it on release because this is a recipe for conflicts

Reply:
Updated.